_posts/2020-10-16-kaitai-struct-v0.9-released.md

```diff
@@ -72,6 +73,7 @@ fixes a lot of bugs and includes quite a few infrastructure improvements.
 * Runtime API changes:
   * Add exceptions `Validation{Not{Equal,AnyOf},{Less,Greater}Than,Expr}Error` inheriting from common ancestor `ValidationFailedError` - thrown on failed validations defined with `valid` or `contents` key ([#435](https://github.com/kaitai-io/kaitai_struct/issues/435))
   * Add method `read_bits_int_le` for parsing little-endian bit-sized integers ([docs](https://doc.kaitai.io/user_guide.html#bit-ints-le))
+  * Allow third-party `process`ors to be used ([#457](https://github.com/kaitai-io/kaitai_struct/issues/457)).
```

Comment:
https://github.com/kaitai-io/kaitai_struct/issues/457 is only some weird Q&A style discussion, I'd rather mark it with the blue label **question**. @GreyCat only mentions the creation of the https://github.com/kaitai-io/kaitai_compress repo, but there is no mention of any changes in KSC or runtime libraries. 

I think this is actually a made-up point: the `kaitai_compress` repo apparently didn't require any such changes. See `git blame` of file [**kaitai_struct_compiler** / shared/.../JavaCompiler.scala](https://github.com/kaitai-io/kaitai_struct_compiler/blame/8d5308d/shared/src/main/scala/io/kaitai/struct/languages/JavaCompiler.scala#L258):

![image](https://user-images.githubusercontent.com/47499687/97495858-724eeb80-1968-11eb-9914-d492a66daced.png)

The commit message states that the last change to that line was done when adding custom `process`ing routines (https://doc.kaitai.io/user_guide.html#custom-process) and it dates back to 2017-07-21. So it's not a change introduced in 0.9, because 0.8 was released on 2018-02-05.